wire-ios-sync-engine/Tests/Source/Integration/SlowSyncTests+NotificationsV3.swift

```diff
@@ -60,9 +61,7 @@ class SlowSyncTests_NotificationsV3: IntegrationTest {
         self.performQuickSync()
 
         // THEN
-        let result = wait(withTimeout: 1) {
-            self.userSession?.applicationStatusDirectory.syncStatus.isSlowSyncing == true
-        }
-        XCTAssertTrue(result, "it should perform slow sync")
+        // ensure it performs slow sync
+        wait(forConditionToBeTrue: self.userSession?.applicationStatusDirectory.syncStatus.isSlowSyncing == true, timeout: 5)
```

Comment:
Quite deep nested test.. maybe the unit test should rather test smaller parts?
But okay for the topic here I understand not to touch it 😁

Reply:
I agree